@stellar/anchor-tests/src/cli.ts

```diff
@@ -72,11 +72,12 @@ const command = yargs
       if (
         (argv.seps.includes(6) ||
           argv.seps.includes(12) ||
+          argv.seps.includes(24) ||
           argv.seps.includes(31) ||
           argv.seps.includes(38)) &&
         !argv.sepConfig
       ) {
-        throw "error: SEP 6, 12, 31 & 38 require a configuration file (--sep-config, -c)";
+        throw "error: SEP 6, 12, 24, 31 & 38 require a configuration file (--sep-config, -c)";
       }
```

Comment:
Is it possible to make the config file optional here? In the UI it sounds like we make it optional and just fail the tests that require the config file, which I think is fine.